src/maintenance/repositories/signing.md

```diff
@@ -0,0 +1,23 @@
+## Signing repositories
+
+Remote repositories **must** be signed, local repositories do not need to be
+signed.
+
+The [xbps-rindex(1)](https://man.voidlinux.org/xbps-rindex.1) tool is used to
+sign repositories. First initialize the repository metadata with signing
+properties (this is only required once).
+
+```
+$ xbps-rindex --sign --signedby "I'm Groot" /path/to/repository/dir
+```
+
+Afterwards sign one or more packages with the following command, newly added
```

Comment:
Afterwards, ...

And, if I am understanding this correctly, it maybe more clear to say:

... with the following command:
```
...
```

Any packages added in the future will also need to be signed.

Reply:
Don't quite agree...

Reply:
Actually, this makes sense, saying "Remember that this only signs the packages you already have. Newly made packages, or changed packages, will not be automatically signed."